aiven/user_config.go

```diff
@@ -25,14 +25,10 @@ func readUserConfigJSONSchema(name string) map[string]interface{} {
 	return jsonObject.(map[string]interface{})
 }
 
-var userConfigSchemas = map[string]map[string]interface{}{}
```

Comment:
I didn't realize the files get served from memory, which certainly helps. In any case it would be easy to write some code that ends up calling GetUserConfigSchema a lot and parsing 100kb string into json each time could have dramatic performance impact. It might be that such code doesn't exist currently but could exist in the future. If the caching is done poorly then feel free to chance but I don't see any reason why we should be parsing immutable json blob more than once in general.

Reply:
I did caching refactoring of this part deserves a separate PR which will be out soon.